README.md

```diff
@@ -57,6 +57,8 @@ Checkout the [good first issue tag](https://github.com/bastislack/highline-frees
 2. Start the development server
 
    run `npm run dev`
+   
+   (or `npm run host` if you want to expose the website to the local network)
```

Comment:
You could also `npm run dev -- --host` to pass the argument to vite.

Reply:
Would that be better? Serious question, are there any benefits or downsides to either having a `host` script or passing the `--host` flag and not having a script?

Reply:
oh no. Keep the `host`script :) 
I just figured the script was added you didn't know about this `-- --$params` notation. If you feel like it would be used a lot, feel free to keep it.

I reckon its useful for checking on Mobile Devices.

Feel free to resolve conversation.

Reply:
Sweet, thanks for the explanation. Seeing the webpage on Mobile was the exact reason I created the command.